.github/CODEOWNERS

```diff
@@ -0,0 +1,15 @@
+  
+# Maps code in repository with maintainers
+# Order is important. The last matching pattern has the most precedence.
+# SEE https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/about-code-owners#example-of-a-codeowners-file
+
+
+# files and folders recursively
+Dockerfile          @elisabettai
+Makefile            @GitHK
```

Comment:
check this!

Reply:
I've cleaned it up. I've also removed the Dockerfile in the parent directory (it was a left over from the previous version we had on GitLab).